templates/pricing/devices.html

```diff
@@ -10,55 +10,58 @@
 <section class="p-strip--suru-topped">
   <div class="u-fixed-width">
     <h1>Ubuntu IoT and device services</h1>
-    <p>Ubuntu is the new standard for embedded Linux development and the intelligent edge. Get ten years of security coverage and a dedicated app store for your smart connected devices.</p>
-    <a href="/support/contact-us" class="p-button--positive js-invoke-modal">Get in touch</a>
+    <p>Ubuntu Core is the standard for embedded Linux deployment. Get ten years of security coverage and fleet management capabilities for your secure connected devices.</p>
+    <a href="/pricing/devices#get-in-touch" class="p-button--positive js-invoke-modal">Get in touch</a>
   </div>
 </section>
 
 <section class="p-strip--light">
   <div class="u-fixed-width">
-    <h2>Get to market quickly with IoT services</h2>
-    <p>Time to market is crucial. Canonical will validate or enable your hardware, package your apps, set up your app store and prepare your device image. It's everything you need to get your device to market fast. App store and over-the-air security updates guaranteed. You can also choose from our extensive list of <a href="/certified?category=Device">certified boards and boxes</a>.</p>
-
-    <h3>IoT and device professional services packages</h3>
-
-    <div class="row u-equal-height u-no-padding">
-      <div class="p-card col-3">
+    <div class="row">
```

Comment:
It seems weird to me to not have an h2 at all here. In addition to seo and accessibility concerns it seems visually inconsistent with every other section on this page which all have clear subheadings. @Sophie-32 @juanruitina what do you guys think?

Reply:
I think it is OK if it's the SmartStart is default plan/option, but I agree it looks weird also because of the gray background. It's also true that "SmartStart" should probably become an H2 instead for a clean accessibility tree. SEO wise I think it's fine: the existing headings were a bit vague and redundant with the page's H1.

Reply:
If its ok for accessibility, I'd prefer to keep in the card since it is a standalone product and then the add-ons are other pieces and I want to show that.